src/DistributedFFT.chpl

```diff
@@ -80,19 +133,50 @@ prototype module DistributedFFT {
     }
   }
 
-  /* Convenience constructor for grids */
+  /***
+      Construct a distributed slab distributed domain.
+      These are currently backed by BlockDist.
+
+      :arg dom: Input 3D rectangular domain
+
+      :returns: Returns a slab-distributed domain.
+  ***/
   proc newSlabDom(dom: domain) where isRectangularDom(dom) {
     if dom.rank !=3 then compilerError("The domain must be 3D");
     const targetLocales = reshape(Locales, {0.. #numLocales, 0..0, 0..0});
     return dom dmapped Block(boundingBox=dom, targetLocales=targetLocales);
   }
 
+  /***
+      Construct a distributed slab distributed domain.
+      These are currently backed by BlockDist.
+
+      :arg sz: Size tuple ``(NX, NY, NZ)``
+
+      :returns: Returns a slab-distributed domain.
+  ***/
   proc newSlabDom(sz) where isHomogeneousTupleType(sz.type) {
     var tup : (sz.size)*range;
     for param ii in 1..sz.size do tup(ii) = 0.. #sz(ii);
     return newSlabDom({(...tup)});
   }
 
+  /***
+      Do the FFT.
+
+      :arg ftType: Type of transform
+      :type ftType: FFTtype
+      :arg src: Input array [XYZ]
+      :arg dest: Output array, transposed [YXZ]
+      :arg signOrKind: Sign (for DFT), or kind (for R2R) of transform.
+
+      Refer to the FFTW documentation for the different
+      signs/kinds.
+
+      Note that if ``ftType`` is ``FFTtype.R2R``, then
+      the ``signOrKind`` argument must be a 3 element array
+      (the type for each dimension separately).
+  ***/
```

Comment:
I missed this before but maybe update array/domain names to match the performant/naive functions (src->Src, dest -> Dest, DestDom->DstDom)

Reply:
Fixed....